examples/runtime-sdk/minimal-runtime/Cargo.toml

```diff
@@ -4,4 +4,4 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-oasis-runtime-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk", tag = "runtime-sdk/v0.4.0" }
+oasis-runtime-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk", tag = "runtime-sdk/v0.8.2" }
```

Comment:
This may require some more changes to build.

Reply:
Locally it builds fine :shrug: It also worked without bumping the runtime-sdk though.

Reply:
Oh, I guess the runtime is really minimal then :-)